aws_cloudtrail_rules/aws_ami_modified_for_public_access.yml

```diff
@@ -19,6 +19,13 @@ Runbook: >
   If not, first modify the AMI to not be publicly accessible then change any sensitive data stored
   in the block devices associated to the AMI (as they may be compromised).
 Reference: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/sharingamis-intro.html
+SummaryAttributes:
+  - userAgent
+  - sourceIpAddress
```

Comment:
Would `p_any_ip_address` also be useful for all events where ip addresses are logged?

Reply:
From what I have seen, requestParameters generally don't have IPs in them 
Since most AWS logs that we get will include a `sourceIpAddress`, I figured this would suffice over `p_any_ip_address`.

Reply:
The normal fields have very specific semantics, so if that is relevant in the context of the rule, you want that. If that is not important then the normalized fields are probably better.  For example, for things like console logins the sourceIP is an important meaning, so that's a good one.